line-bot-api.gemspec

```diff
@@ -2,21 +2,27 @@ version = File.read("#{__dir__}/lib/line/bot/version.rb").match(/VERSION *= *['"
 raise "Invalid version format <#{version}>" unless version.match?(/\A\d+\.\d+\.\d+(?:[+-][\w.-]+)?\z/)
 
 Gem::Specification.new do |spec|
-  spec.name          = "line-bot-api"
-  spec.version       = version
-  spec.authors       = ["LINE Corporation"]
-  spec.email         = ["kimoto@linecorp.com", "todaka.yusuke@linecorp.com", "masaki_kurosawa@linecorp.com"]
```

Comment:
We are very grateful to past maintainers even who are no longer involved in development, but there is no need to include their email addresses anymore. Their contributions are documented in the commit log and this repository.